prow/cmd/deck/static/pr-script.js

```diff
@@ -1020,7 +1141,7 @@ function jobStatus(builds) {
  * @param queries
  * @return {number}
  */
-function isAbleToMerge(queries) {
+function hasResolvedLabels(queries) {
```

Comment:
This function could just return a bool (making its behavior less confusing) if it took a single query instead of the list.